<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ed_cta_lote</class>
 <widget class="QDialog" name="ed_cta_lote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edici√≥n de cuenta</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="externo_label">
       <property name="text">
        <string>Externo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="busca_externo_pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../keme.qrc">
         <normaloff>:/iconos/graf/xmag.png</normaloff>:/iconos/graf/xmag.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="externo_lineEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="datos_externo_pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../keme.qrc">
         <normaloff>:/iconos/graf/edit-find-mail.png</normaloff>:/iconos/graf/edit-find-mail.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quita_externo_pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../keme.qrc">
         <normaloff>:/iconos/graf/borraelem.png</normaloff>:/iconos/graf/borraelem.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="descrip_externo_lineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cuenta:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cuentalineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="ctapushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../keme.qrc">
         <normaloff>:/iconos/graf/xmag.png</normaloff>:/iconos/graf/xmag.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="descrip_label">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>- -</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="cuota_label">
         <property name="text">
          <string>Cuota (%):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="tipolineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="importe_label">
         <property name="text">
          <string>Importe   :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="importelineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>ed_cta_lote</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aceptarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>ed_cta_lote</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
